modules/creating-serverless-apps-kn.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// serverless/serving-creating-managing-apps.adoc
+
+[id="creating-serverless-apps-kn_{context}"]
+= Creating serverless applications using the kn CLI
+
+.Prerequisites
+* Installed the {ServerlessOperatorName} and Knative Serving.
+* Installed `kn` CLI.
+
+.Procedure
+
+. Create the Knative service by entering:
++
+----
+$ kn service create <SERVICE-NAME> --image <IMAGE> --env <KEY=VALUE>
+----
++
+.Example
++
+----
+kn service create hello --image gcr.io/knative-samples/helloworld-go --env TARGET=Knative
+
+Creating service 'hello' in namespace 'default':
+
+  0.271s The Route is still working to reflect the latest desired specification.
+  0.580s Configuration "hello" is waiting for a Revision to become ready.
+  3.857s ...
+  3.861s Ingress has not yet been reconciled.
+  4.270s Ready to serve.
+
+Service 'hello' created with latest revision 'hello-bxshg-1' and URL:
+http://hello.default.apps-crc.testing
```

Comment:
http://hello-default.apps-crc.testing 

( `hello-default`  instead of `hello.default` , since 1.7.0, sorry for missing this in the first round... )

Reply:
np, thank you!